packages/mgt/src/components/mgt-tasks/mgt-tasks.ts

```diff
@@ -612,7 +612,9 @@ export class MgtTasks extends MgtTemplatedComponent {
     } as ITask;
 
     this._newTaskBeingAdded = true;
-    await ts.addTask(newTask);
+    const task = await ts.addTask(newTask);
+    this.fireCustomEvent('taskAdded', task);
```

Comment:
Make sure to add the `@fires` doc at the top of the class to document the events fired by the component. This is also used by mgt-react to generate the types.

Reply:
Good catch! Updated